src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -388,6 +394,22 @@ public function newLine($count = 1)
     }
 
     /**
+     * Sets the input stream used by the SymfonyQuestionHelper.
+     *
+     * @param resource $stream
+     *
+     * @throws InvalidArgumentException If the given stream is not a resource
```

Comment:
```

    /**
     * Sets the input stream to read from when interacting with the user.
     *
     * This is mainly useful for testing purpose.
     *
     * @param resource $stream The input stream
     *
     * @throws InvalidArgumentException In case the stream is not a resource
     */
```